package.json

```diff
@@ -10,25 +10,16 @@
     "test": "mocha test/*.js",
     "tdd": "nodemon -x npm -i node_modules/ -- test",
     "note1": "we can't have nice things. prepublish also runs on npm install https://github.com/npm/npm/issues/6394 in-publish hacks around this",
-    "prepublish": "in-publish && source ./scripts.sh && npm prune && npm run nsp && npm run gitPush || in-install",
+    "prepublish": "in-publish && source ./scripts.sh && npm prune && npm run gitPush || in-install",
     "lint": "eslint .",
     "requireGitClean": "source ./scripts.sh && git_require_clean_work_tree",
-    "nsp": "nsp check",
```

Comment:
Could/should we run `npm audit` as well?

Reply:
That is the replacement for the `nsp check`. We do this internally. I would say the only way this would be relevant is if this starts getting traction to the point where it is actively being worked on and then it could make sense to make it part of the build process. With that said I don't have a whole lot of opinion :D